.github/workflows/compress-images.yml

```
@@ -0,0 +1,38 @@
+          fetch-depth: 0
+      - name: Compress Images
+        id: calibre
+        uses: calibreapp/image-actions@main
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          compressOnly: true
+          ignorePaths: "node_modules/**,ios/**,android/**"
+
+      - name: Create Pull Request
+        if: steps.calibre.outputs.markdown != ''
+        uses: peter-evans/create-pull-request@v3
```

actionlint

the runner of "peter-evans/create-pull-request@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)